github-actions.md

```diff
@@ -0,0 +1,24 @@
+# GitHub Actions
+
+Github actions should be used for Continuous Integration. Actions are typically used to enforce checks before merging branches into main (such as formatting, linting and testing), as well as to publish libraries or [deploy](continuous-deployment.md) build artifacts (via Riff-Raff).
+
+## Specifying versions for actions you use
+
+Github Actions workflows can invoke other actions via `uses` steps. When specifying the version to use, one can typically:
+
+- Reference via a Git tag e.g `uses: actions/checkout@v4`. These correspond to tagged releases on Github - see [actions/checkout/releases](https://github.com/actions/checkout/releases) for examples.
+- Reference via a Git Commit SHA e.g. `uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1`.
+
+**We recommend specifying the version of Github Actions workflows as a commit SHA**.
```

Comment:
Perhaps we should include some information on how to find the SHA digest for a particular release?

I personally go to the releases page and click the SHA digest in the panel on the left (below) then copy the full commit SHA out of the URL. This might not be the best way though. 

<img width="195" alt="image" src="https://github.com/guardian/recommendations/assets/5560113/ed6f7db5-7b89-49ac-acd4-7287ef274335">

Reply:
Good idea, I'll nab your screenshot while I'm at it